WordPress/wporg-news-2021
# Single post view

Adds block templates and template parts, tweaks the theme.json file, and adds some rules to the stylesheet to create the single post view as depicted in the mockup in #17.

This includes a responsive layout for the post title/meta area and the main post content column. The post content also accommodates wide and full alignment on individual blocks.

This doesn't include the global header or footer (see #6). It also doesn't include the social sharing links or like button shown in the mockup, because those elements are not yet available as site blocks. (There's [an issue](https://github.com/Automattic/jetpack/issues/19659) for the Like button, couldn't find one for social sharing icons.)

The mockup doesn't include a comment section, and it appears that the News site does not currently have comments enabled, so that has also been omitted.

Fixes #17 

## To test

1. The post used in the mockup is also included in the test content that gets added to the local dev environment for this repo, so you can easily compare the two.
2. Try changing the alignment of some blocks in the post to Wide and Full to see how those work.
3. Try changing the viewport size to find the responsive breakpoints.
